spyder/utils/vcs.py

```diff
@@ -30,7 +30,8 @@
     'name': 'Git',
     'rootdir': '.git',
     'actions': dict(
-        commit=( ('git', ['gui' if os.name == 'nt' else 'cola']), ),
+        commit=( ('git', ['gui']),
+                 ('cola', []), ),
```

Comment:
```suggestion
        commit=( ('git-cola', []),
                 ('git', ['gui']), ),
```

Prefer [`git-cola`](https://github.com/git-cola/git-cola) (the right command for it) if it's installed because the interface is nicer than `git gui`.